scripts/globals/items/a_loutrance.lua

```diff
@@ -0,0 +1,25 @@
+-----------------------------------------
+-- ID: 18041
+-- Item: A l'Outrance
+-- Additional Effect vs. beasts: Darkness Damage
+-----------------------------------------
+require("scripts/globals/magic")
+require("scripts/globals/msg")
+require("scripts/globals/status")
+-----------------------------------------
+
+function onAdditionalEffect(player, target, damage)
+    local dmg = player:getStat(dsp.mod.INT) - target:getStat(dsp.mod.INT)
```

Comment:
Add effect melee items don't calculate this way. ranged items do.

Reply:
also still making more work for me for the global thing..

Reply:
Ascalon specifically said it uses dMND for damage, but who knows. I'll just remove all the additional effect ones for now.

Reply:
could be special case, could be shitWikiSays™

I did not see any diff from stacking INT or MND on:
Hydro Baghnakhs, Inferno Axe, Burning Cesti, Burning Cesti, Spark Spear, Spark Spear +1, Estramacon, Fire Sword, Pixie Mace, Holy Mace, Holy maul, Holy Maul +1, Cougar Baghnakhs, Shiva's Claws, Sonic Knuckles, or Fuscina

I also was looking for player lv vs mob lv to modify damage. I saw no difference.

So I thought either it doesn't effect these or the range is narrow and capped to either a maximum or minimum. I tested for level difference to. Neither my level nor the mobs level had any visible impact. Certain weapons hit notably harder and in a wider random range. Very possible there are diff rules for certain weapons and also very possible that someone got a big hit and didn't really parse a diff (I haven't seen any data corroborating Ascalon).

Reply:
p.s.
in the event we can show ascalon DOES work that way, I'm inclined to think all the "vs <family>" will be that way as well. All the ones I tested were not vs any mob family,